frontend/src/app/components/MainInstallButton.js

```diff
@@ -12,6 +12,13 @@ export default class MainInstallButton extends React.Component {
     };
   }
 
+  install() {
+    const { eventCategory, hasAddon } = this.props;
+    if (hasAddon) { return; }
+    this.setState({ isInstalling: true });
```

Comment:
This is the sort of thing I'd like to get rid of in #1367; there's no reason this value shouldn't be on the Redux store and available for other components to access.